src/molecules/accessible-list/accessible-list.test.tsx

```diff
@@ -1,7 +1,81 @@
-// import React from "react";
-// import { render } from "@testing-library/react";
-// import { AccessibleList } from "./accessible-list";
+import React, { Component } from "react";
+import faker from "faker";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { AccessibleList } from "./accessible-list";
+import { KeyboardKeys } from "../../constants/keyboard-keys";
 
 describe("AccessibleList", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/19", () => {});
+    test("when default props, renders items", () => {
+        // Arrange
+        const expected = faker.random.word();
+
+        // Act
+        const { getByText } = render(
+            <AccessibleList focusFirstItem={false}>
+                <button>{expected}</button>
+            </AccessibleList>
+        );
+
+        // Assert
+        expect(getByText(expected)).not.toBeNil();
+    });
+
+    test("when onEsc set, calls handler upon escape pressed", () => {
+        // Arrange
+        let isChecked = false;
+        const buttonText = faker.random.word();
+        const onEscHandler = () => {
+            isChecked = true;
+        };
+
+        // Act
+        const { getByText } = render(
+            <AccessibleList focusFirstItem={true} onEsc={onEscHandler}>
+                <button>{buttonText}</button>
+            </AccessibleList>
+        );
+        fireEvent.keyDown(getByText(buttonText), { key: KeyboardKeys.Escape });
+
+        // Assert
+        expect(isChecked).toBeTrue();
+    });
+
+    test("when invalid react element, does not render in list", () => {
+        // Arrange
+        const buttonText = [faker.random.word(), faker.random.word()];
+
+        // Act
+        const { container } = render(
+            <AccessibleList focusFirstItem={true}>
+                <button>{buttonText[0]}</button>
+                {null}
+                {undefined}
+                <button>{buttonText[1]}</button>
+            </AccessibleList>
+        );
+
+        // Assert
+        expect(container.innerHTML).not.toContain("null");
```

Comment:
I wonder if this is actually testing what we think we're testing. I've seen `null` used as a "short-circuit" return in a component that is not meant to render due to a certain condition, but that doesn't mean it renders the text "null" in the DOM. Is there a way to easily check for number of children from the returned `container`? Or perhaps that every child is of type `button`, since that's the only valid element we're using as children?

Reply:
@brandongregoryscott I removed the check for valid react element to test and it does break when passing in null and undefined. I changed the Assert to check and make sure the total child nodes being rendered is a value of 2 since we are only creating 2 valid nodes. I think this would address your concerns, but please let me know if you would like any other changes.

Reply:
Thanks for double checking that. Weird, I would not expect `null` or `undefined` to be in the inner HTML. Sorry for the late response, this looks good to go 👍